conda_build_config.yaml

```diff
@@ -0,0 +1,30 @@
+c_compiler:         # [win]
+  - vs2008          # [win]
+  - vs2015          # [win]
+cxx_compiler:       # [win]
+  - vs2008          # [win]
+  - vs2015          # [win]
+hdf5:
+  - 1.8
+  - 1.10
+macos_min_version:
+  - 10.9
+numpy:
```

Comment:
no.  The way it is now implements @jjhelmus' nice scheme of having numpy be forwards compatible